sdk/resourcemanager/azure-resourcemanager/src/samples/java/com/azure/resourcemanager/ThreadPoolSamples.java

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.netty.NettyAsyncHttpClientBuilder;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import reactor.netty.resources.ConnectionProvider;
+import reactor.netty.resources.LoopResources;
+
+import java.time.Duration;
+import java.util.concurrent.ForkJoinPool;
+
+/**
+ * Code samples for the THREAD_POOL_AND_CONNECTION_POOL.md
+ */
+public class ThreadPoolSamples {
+
+    public AzureResourceManager azureClientThreadPool(String subscriptionId) {
+        NettyAsyncHttpClientBuilder singletonHttpClientBuilder = new NettyAsyncHttpClientBuilder();
+        // BEGIN: readme-sample-azureClientConnectionPool
+        singletonHttpClientBuilder
+            // Connection pool configuration.
+            // Official Reactor Netty documentation for defaults: https://projectreactor.io/docs/netty/release/reference/#_connection_pool_2
+            .connectionProvider(
+                ConnectionProvider.builder("connection-pool")
+                    // By default, HttpClient uses a “fixed” connection pool with 500 as the maximum number of active channels
```

Comment:
nit, avoid unnecessary unicode in code and md. here `“fixed”`

Reply:
OK, didn't realize that..